src/webgpu/api/operation/vertex_state/correctness.spec.ts

```diff
@@ -552,10 +592,14 @@ struct VSOutputs {
 
       for (const attrib of buffer.attributes) {
         const formatInfo = kVertexFormatInfo[attrib.format];
+        const size =
```

Comment:
Why not just `formatInfo.byteSize`?

Reply:
When using `formatInfo.byteSize` as suggested, http://127.0.0.1:8080/standalone/?q=webgpu:api,operation,vertex_state,correctness:vertex_format_to_shader_format_conversion:format=%22rgb10a2%22;shaderComponentCount=1 returns the error below:
```
EXCEPTION: subcase: slot=0;shaderLocation=0
offset is out of bounds
RangeError: subcase: slot=0;shaderLocation=0
offset is out of bounds
    at Uint8Array.set (<anonymous>)
    at memcpy (http://127.0.0.1:8080/out/common/util/util.js:424:30)
    at VertexStateTest.interleaveVertexDataInto (http://127.0.0.1:8080/out/webgpu/api/operation/vertex_state/correctness.spec.js:529:7)
    at VertexStateTest.createVertexBuffers (http://127.0.0.1:8080/out/webgpu/api/operation/vertex_state/correctness.spec.js:596:14)
    at VertexStateTest.runTest (http://127.0.0.1:8080/out/webgpu/api/operation/vertex_state/correctness.spec.js:624:32)
    at RunCaseSpecific.fn (http://127.0.0.1:8080/out/webgpu/api/operation/vertex_state/correctness.spec.js:649:5)
```

Reply:
With latest fix below,  I get another error (better I guess):
```diff
           return {
             shaderBaseType: 'f32',
-            testComponentCount: data.length,
+            testComponentCount: data.flat().length,
             expectedData,
             vertexData,
             floatTolerance: 0.1 / 1023, // TODO: Change?
```
```
EXPECTATION FAILED: subcase: slot=0;shaderLocation=0
Array had unexpected contents at indices 4 through 4940.
 Starting at index 3:
   actual == 0x: 00 00 00 00 00 00 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 01 00 ...
   failed ->        xx          xx                      xx          xx                      xx          xx                ...
 expected ==     00 01 00 00 00 01 00 00 00 01 00 00 00 01 00 00 00 01 00 00 00 01 00 00 00 01 00 00 00 01 00 00 00 01 00 ...
Error
    at VertexStateTest.eventualAsyncExpectation (http://127.0.0.1:8080/out/common/framework/fixture.js:195:24)
    at VertexStateTest.expectGPUBufferValuesPassCheck (http://127.0.0.1:8080/out/webgpu/gpu_test.js:416:10)
    at VertexStateTest.expectGPUBufferValuesEqual (http://127.0.0.1:8080/out/webgpu/gpu_test.js:432:10)
    at VertexStateTest.expectGPUBufferRepeatsSingleValue (http://127.0.0.1:8080/out/webgpu/gpu_test.js:476:12)
    at VertexStateTest.expectSingleColor (http://127.0.0.1:8080/out/webgpu/gpu_test.js:643:10)
    at VertexStateTest.submitRenderPass (http://127.0.0.1:8080/out/webgpu/api/operation/vertex_state/correctness.spec.js:311:10)
    at VertexStateTest.runTest (http://127.0.0.1:8080/out/webgpu/api/operation/vertex_state/correctness.spec.js:625:10)
    at RunCaseSpecific.fn (http://127.0.0.1:8080/out/webgpu/api/operation/vertex_state/correctness.spec.js:649:5)
```